modules/400-descheduler/crds/deschedulers.yaml

```diff
@@ -0,0 +1,584 @@
+                                  type: object
+                                  x-kubernetes-map-type: atomic
+                                namespaces:
+                                  properties:
+                                    exclude:
+                                      items:
+                                        type: string
+                                      type: array
+                                    include:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                podLifeTime:
+                                  properties:
+                                    maxPodLifeTimeSeconds:
+                                      type: integer
+                                    podStatusPhases:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                thresholdPriority:
+                                  format: int32
+                                  type: integer
+                                thresholdPriorityClassName:
+                                  type: string
+                              type: object
+                          type: object
+                        removeDuplicates:
+                          properties:
+                            params:
+                              properties:
+                                namespaces:
+                                  properties:
+                                    exclude:
+                                      items:
+                                        type: string
+                                      type: array
+                                    include:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                nodeFit:
+                                  type: boolean
+                                removeDuplicates:
+                                  properties:
+                                    excludeOwnerKinds:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                thresholdPriority:
+                                  format: int32
+                                  type: integer
+                                thresholdPriorityClassName:
+                                  type: string
+                              type: object
+                          type: object
+                        removeFailedPods:
```

Comment:
An example how it can be.
```
podLifeTime:
     enabled: true
     params:
         maxPodLifeTimeSeconds: 86400
         states:
         - "Pending"
         - "PodInitializing"
```

Reply:
Done.